README.md

```diff
@@ -1,3 +1,29 @@
 # fabric8-gemini-server
 Fabric8-server analytics powered services to initiate and report the readiness of
 all registered services
+
+### ENDPOINTS
+
+#### POST: /api/v1/register
+```
+{
+	"git_url" : <some_url>,
+	"git_sha" : <some_sha>,
+	"email_ids" : <some_email>
```

Comment:
To be explicit here,

Value to `email_ids` key, is comma separated string.
Example: `"nshaikh@redht.com,samuzzal@redhat.com"`. To yield the email id list, `string.strip().split(",")` should give desired result.

Reply:
@samuzzal-choudhury, do we need a list for email-ids?

Reply:
@GeetikaBatra yes. email ids is a list.